internal/vulnstore/dastore/get.go

```diff
@@ -54,63 +54,161 @@ type Report struct {
 	Result Result `json:"result"`
 }
 
+type Request struct {
+	Ecosystem string `json:"ecosystem"`
+	Package string `json:"package"`
+	Version string `json:"version"`
+}
+
+func call(req []Request,ids []string, results map[string][]*claircore.Vulnerability, wg *sync.WaitGroup){
+
+	fmt.Println("Inside call")
+   jsonValue, _ := json.Marshal(req)
+   // fmt.Println(jsonValue)
+	response, err := http.Post("https://f8a-analytics-2445582058137.production.gw.apicast.io:443/api/v1/component-analyses/?user_key=9e7da76708fe374d8c10fa752e72989f", "application/json", bytes.NewBuffer(jsonValue))
+	 if err != nil {
+		 fmt.Printf("The HTTP request failed with error %s\n", err)
+	 } else {
+		 fmt.Println("HII")
+		 var da_response []Report
+ 
+		
+		 data, _ := ioutil.ReadAll(response.Body)
+	//	 fmt.Println(len(data),id)
+ 
+		  err = json.Unmarshal(data, &da_response)
+		 if err != nil {
+			 fmt.Println(err)
+		 }
+	//     fmt.Println(len(da_response))
+ 
+	  //   fmt.Println(da_response[0])
+ 
+	fmt.Println("Printing length of data ",len(data))
+
+
+	  for i:=0;i<len(ids);i++{
+
+          
+		 	          if len(da_response[i].Result.Recommendation.ComponentAnalysis.Cve)>0 {
+
+					//	var v *claircore.Vulnerability
+						fmt.Println("Printing cve array ",da_response[i].Result.Recommendation.ComponentAnalysis.Cve)
+                         var vulnArray []*claircore.Vulnerability
+
+				//		for k:=0;k<len(da_response[i].Result.Recommendation.ComponentAnalysis.Cve);k++{
+					//		fmt.Println("Inside ", da_response[i].Result.Recommendation.ComponentAnalysis.Cve[k].Idd)
+						  vulnArray=append(vulnArray,&claircore.Vulnerability{
+						ID:                 ids[i],
+							Updater:            "",
+				 			Name:               da_response[i].Result.Recommendation.ComponentAnalysis.Cve[0].Idd,
+							Description:        da_response[i].Result.Recommendation.Message,
+	 					Links:              "",
+		 	 			Severity:           fmt.Sprint(da_response[i].Result.Recommendation.ComponentAnalysis.Cve[0].Cvss),
+							// NormalizedSeverity: "",
+				 			FixedInVersion:     da_response[i].Result.Data[0].Cvee.Fixed_in[0],
+			 			Package: &claircore.Package{ID: "0",
+				 				Name:    "xyz",
+								Version: "v0.0"},
+						Dist: &claircore.Distribution{},
+						Repo: &claircore.Repository{},
+				  })
+
+					  results[ids[i]] = vulnArray
+					}
+				 
+			   }
+	 }
+ 
+	 wg.Done()
+ 
+  }
```

Comment:
Please format the code according to the coding style of claircore.

Remove all prints and use appropriate log statements.